hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/autocleanup/AutoActionCleanupTest.java

```diff
@@ -123,6 +123,7 @@ public void canceledAndFailedActionsAreCleanedUp() {
 
         assertThat(actionRepository.count()).isEqualTo(3);
 
+        Thread.sleep(1000);
```

Comment:
use Awaitility /waitUntilAsserted instead

Reply:
Thread.sleep was just a temporary test in order to find out if there is some kind of async issue with this test - I remove it.